catroid/src/main/java/org/catrobat/catroid/io/SoundManager.java

```diff
@@ -42,6 +44,7 @@
 
 	private final List<MediaPlayer> mediaPlayers = new ArrayList<MediaPlayer>(MAX_MEDIA_PLAYERS);
 	private float volume = 70.0f;
+	private List<String> soundFiles = new ArrayList<>(MAX_MEDIA_PLAYERS);
 
 	public SoundManager() {
```

Comment:
This line seems not right. Could you please check why this is public? (I know you didn't write this line ^^)
(if it's only used in tests => visible for testing and hopefully, there is no occurrence in the code)

Reply:
Or if it's in the tests => mock. Thank you!